extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java

```diff
@@ -2,15 +2,23 @@
 
 import com.grack.nanojson.JsonArray;
 import com.grack.nanojson.JsonObject;
+
+import org.omg.PortableServer.LIFESPAN_POLICY_ID;
```

Comment:
uhm? Please remove this import.

Reply:
Oh, sorry, I probably accidentally pressed Alt+Enter.